lib/Agrammon/Model/Result.pm6

```diff
@@ -1,9 +1,44 @@
 use v6;
 
+use Agrammon::LanguageParser;
+
 class Agrammon::Model::Result {
     has Str $.name;
     has Str $.type;
-    has  %.selector{Str};
-    has %.data;
+    has %.selector{Str};
+    has @!data-order;
+    has %!data-lookup;
     has Str $._order;
```

Comment:
I'm a bit curious why the `_` here

Reply:
For some reason I had named it that way in the old version. As there is no occurance of it in the latest `.nhd` files, I changed it to `order`. We might not even need that anymore, but if we do, we can always use the more sensible name.
I also noticed that in two more `.js` files which also will need more refactoring/fixing soon, but for this PR I just changed the `_order`.